cli/extract_census_template.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os
+import subprocess
+
+from app.survey_schema import SurveySchema
+from app.schema_translation import SchemaTranslation
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Extract a combined translation template from census json schemas")
+
+    parser.add_argument(
+        'RUNNER_SCHEMA_DIRECTORY',
+        help="The path to the runner directory from which data will be extracted"
+    )
+
+    parser.add_argument(
+        'OUTPUT_DIRECTORY',
+        help="The destination directory for the translation template"
+    )
+
+    args = parser.parse_args()
+
+    if not os.path.isdir(os.path.join(args.RUNNER_SCHEMA_DIRECTORY)):
+        print("Not a valid runner schema directory")
```

Comment:
The error message here is a bit misleading - isdir only checks if the directory exists, not whether it is a valid runner schema directory.